media-libs/alembic/patches/alembic-1.7.4.patchset

```diff
@@ -0,0 +1,62 @@
+From 10b2076de96a69df6cf55c27feaeaeaf6e14b9a2 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Thu, 7 Dec 2017 19:03:18 +0100
+Subject: [PATCH] Haiku build fix
+
+---
+ cmake/Modules/FindIlmBase.cmake            | 4 ++++
+ lib/Alembic/AbcCoreOgawa/StreamManager.cpp | 8 ++++++--
+ 2 files changed, 10 insertions(+), 2 deletions(-)
+
+diff --git a/cmake/Modules/FindIlmBase.cmake b/cmake/Modules/FindIlmBase.cmake
+index dac8659..3712e84 100644
+--- a/cmake/Modules/FindIlmBase.cmake
++++ b/cmake/Modules/FindIlmBase.cmake
+@@ -85,6 +85,7 @@ SET(_ilmbase_SEARCH_DIRS
+     /opt/csw
+     /opt
+     /usr/freeware
++    /boot/system
+ )
+ 
+ FIND_PATH(ILMBASE_INCLUDE_DIR
+@@ -95,6 +96,9 @@ FIND_PATH(ILMBASE_INCLUDE_DIR
+   PATH_SUFFIXES
+     include
+     include/OpenEXR
++    develop/headers/OpenEXR
++    develop/headers/x86/OpenEXR
```

Comment:
Hmm, that's not recommended. There are better ways to pass the includes dir.
That said, if this does not hurt anyone else, then I can sleep with it ;-)